src/plugin/index-more/src/java/org/apache/nutch/indexer/more/MoreIndexingFilter.java

```diff
@@ -316,6 +324,39 @@ public void setConf(Configuration conf) {
         LOG.error(org.apache.hadoop.util.StringUtils.stringifyException(e));
       }
     }
+    
+    URL dateStylesResource = conf.getResource("date-styles.txt");
+    if (dateStylesResource == null) {
+      LOG.warn("Can't find resource: date-styles.txt");
+    } else {
+      try {
+        List lines = FileUtils.readLines(new File(dateStylesResource.getFile()));
+        for (int i = 0; i < lines.size(); i++) {
+          String dateStyle = (String) lines.get(i);
+          
+          if (StringUtils.isBlank(dateStyle)) {
+            lines.remove(i);
```

Comment:
Since we don't need the line number for the next stages, perhaps could skip the line if it is empty, `\n`, or a comment (`#`). The idea would be to avoid modifying the counter and if we reach the end of the block then we know that the line has information that should be kept.